### PR TITLE
SALTO-6209: Salesforce Billing Triggers Pre & Post Deploy Actions

### PR DESCRIPTION
**Pull request by tamtamirr**
Salesforce Billing Triggers Pre & Post Deploy Actions

---

_Release Notes_: 
Salesforce Adapter:
- Added pre & post deploy actions that guide users to disable their billing triggers for `CPQ Billing Deployments`. 

---
_User Notifications_: 
_None_

### CODE REVIEW

**Comment by hadard** [comment]
@yelly @tomermevorach any news on this? we want to take it to prod on Wednesday's deploy

**Comment by yelly** [comment]
@hadard Tomer and Tamir came to some agreement, I'm not sure what. We'll get it merged tomorrow.

File: packages/salesforce-adapter/test/change_validators/cpq_billing_triggers.test.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  let billingMockType: ObjectType
+
+  beforeEach(() => {
+    billingMockType = createCustomObjectType(
+      `${BILLING_NAMESPACE}__MockType__c`,
+      {},
+    )
+  })
+
+  describe('when theres no change of Billing records', () => {
```
Comment (yelly) [inline packages/salesforce-adapter/test/change_validators/cpq_billing_triggers.test.ts:32]:
```suggestion
  describe("when there's no change of Billing records", () => {
```

File: packages/salesforce-adapter/test/change_validators/cpq_billing_triggers.test.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        { Id: '123', Name: 'Test' },
+      )
+      const changeErrors = await cpqBillingTriggersChangeValidator([
+        toChange({ after: dataInstance }),
+      ])
+      expect(changeErrors).toBeEmpty()
+    })
+  })
+
+  describe('when theres change of Billing records', () => {
```
Comment (yelly) [inline packages/salesforce-adapter/test/change_validators/cpq_billing_triggers.test.ts:46]:
```suggestion
  describe("when there's a change of Billing records", () => {
```

**Review by yelly — APPROVED** [review]
Just a couple small nits. Good to merge after fixing.

**Pull request merged**